src/webapp/src/characters/charactersSlice.ts

```diff
@@ -0,0 +1,63 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+interface StateData {
```

Comment:
I will avoid using **data** as a name! You are totally right! When you mention those things it becomes so clear and obvious in my mind that I don't know what I was thinking whenever I choose those names. 

Thanks for pointing these little things with so much patience man. You cannot imagine how much I am learning in each of your reviews! 

Since we started doing the project, whenever I am doing tutoring or helping the other students in their projects I am a machine of copying your way of communicating, and your advice about coding! I always give you the credit, don't worry! :D